package.json

```diff
@@ -21,7 +21,7 @@
   },
   "homepage": "https://github.com/webpack-contrib/expose-loader",
   "peerDependencies": {
-    "webpack": "^2.0.0 || ^3.0.0"
+    "webpack": "*"
```

Comment:
```diff
-  "webpack": "^2.0.0 || ^3.0.0"
+ "webpack": "^2.0.0 || ^3.0.0 || ^4.0.0-alpha.0"
```

Reply:
What do you think about removing this peerDependency altogether? I noticed that a bunch of our webpack-related dependencies have gone that route.

Reply:
I can't decide that, but I agree that the the `peerDependency` is often just noise atm and maybe should just be set on loaders/plugins that really need it. On the other hand we are trying to bring in consistency across `webpack-contrib` about things like that, but simply aren't there yet :)

Reply:
cc @d3viant0ne